fontforgeexe/collabclientui.c

```diff
@@ -471,30 +471,32 @@ gpointer getSessionCallbacksObject()
 void collabclient_notifySessionJoining( cloneclient_t *cc, FontViewBase* fv )
 {
 #ifdef BUILD_COLLAB
-    
-    printf("AAA collabclient_notifySessionJoining() fv:%p\n", fv);
+
+    printf("AAA collabclient_notifySessionJoining() cc: %p, fv:%p\n", cc, fv);
```

Comment:
@jtanx, I think that I meant to say that we ought to have an error message in the event that cc is null in the other function. Otherwise we might silently miss an error there.